pageserver/src/tenant/remote_timeline_client/index.rs

```diff
@@ -233,12 +249,14 @@ impl IndexPart {
 #[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Default)]
 pub struct IndexLayerMetadata {
     pub(super) file_size: Option<u64>,
+    pub(super) holes: Option<Vec<Hole>>,
```

Comment:
The `IndexPart::LATEST_VERSION` should be bumped from 1 to 2, even though it was missed when removing the `missing_layers`.

Could you add an example of this new format with holes populated, similar to `v1_indexpart_is_parsed`?

Reply:
Done

Reply:
I think you replaced the `v1_indexpart_is_parsed`? Please just copy a new test for it. The idea is to have "images" of all these indexparts we *must* support in the tests. We still need to support v1 indexparts.

Reply:
Ok

Reply:
This looks good, resolving.